include/alpaka/kernel/TaskKernelCpuOmp2Blocks.hpp

```diff
@@ -156,15 +169,17 @@ namespace alpaka
                 *static_cast<WorkDivMembers<TDim, TIdx> const*>(this),
                 blockSharedMemDynSizeBytes);
 
-            // NOTE: schedule(static) does not improve performance.
+            auto oldSchedule = omp::getSchedule();
+            omp::setSchedule(schedule);
 #    if _OPENMP < 200805 // For OpenMP < 3.0 you have to declare the loop index (a signed integer) outside of the loop
                          // header.
             std::intmax_t iNumBlocksInGrid(static_cast<std::intmax_t>(numBlocksInGrid));
             std::intmax_t i;
+            // OpenMP 2.0 does not support schedule(runtime), stick to guided
```

Comment:
Are you sure? The OpenMP 2.0 specification does mention `schedule(runtime)`. See https://www.openmp.org/wp-content/uploads/cspec20.pdf, page 13. Is this maybe confused with `schedule(auto)`?